AcceptanceTests/AcceptanceTests.swift

```diff
@@ -183,11 +183,32 @@ class AcceptanceTests: QuickSpec {
             
             context("with the 'help' command") {
                 they("have the list of available commands displayed to them") {
+                    expect(self.muterHelpOutput).to(contain("🔎 Automated mutation testing for Swift 🕳️"))
```

Comment:
This stuff doesn't account for order. Is the string 100% static each run? If so, we could do an equality check with a big ol' multiline string literal. That would at least alert us if the format changed.